packages/proxy-container/README.md

```diff
@@ -48,6 +48,9 @@ const rest = new REST({
 });
 ```
 
+**Do note that you should not use the proxy with multiple bots. We cannot guarantee you won't hit rate limits.
```

Comment:
This could be clarified that there's no problem using [multiple] proxies for multiple bots, but rather that there is when the same proxy is reused for multiple bots.
```suggestion
**Do note that you should not use the same proxy with multiple bots. We cannot guarantee you won't hit rate limits.
```